cpp/modmesh/buffer/ConcreteBuffer.hpp

```diff
@@ -213,14 +213,13 @@ class ConcreteBuffer
         , m_nbytes(other.m_nbytes)
         , m_data(allocate(other.m_nbytes))
     {
+        m_begin = m_data.get(); // override m_begin and m_end once we have the data
```

Comment:
need to initialize before the following `size()` call.

Reply:
At some places you wrote "overwrite" and here you write "override" `m_begin` and `m_end`.  Please word consistently.